packages/shared/src/components/cards/FeedItemContainer.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { forwardRef, HTMLAttributes, ReactElement, Ref } from 'react';
+import { Post } from '../../graphql/posts';
+import { Card, ListCard } from './Card';
+import { RaisedLabel, RaisedLabelType } from './RaisedLabel';
+import ConditionalWrapper from '../ConditionalWrapper';
+import styles from './Card.module.css';
+
+interface FlagProps extends Pick<Post, 'trending' | 'pinnedAt'> {
+  listMode?: boolean;
+}
+
+interface FeedItemContainerProps extends HTMLAttributes<HTMLDivElement> {
+  flagProps: FlagProps;
+}
+
+function FeedItemContainer(
+  { flagProps, ...props }: FeedItemContainerProps,
+  ref?: Ref<HTMLElement>,
+): ReactElement {
+  const { listMode, pinnedAt, trending } = flagProps;
+  const Component = listMode ? ListCard : Card;
+  const type = pinnedAt ? RaisedLabelType.Pinned : RaisedLabelType.Hot;
+  const description =
```

Comment:
Since we only need this in one place now, there's no need to create a component. Hence, applied here manually as this will be the container of all Feed items.

Will remove unused components.